### PR TITLE
Updates sept 23

### PR DESCRIPTION
**Pull request by neha-bhargav**
Previous changes + Merged changes from Master

### CODE REVIEW

File: _data/team_members.yml
Comment (sid18996) [inline _data/team_members.yml:1]: Siddhartha is co-advised right? @neha-bhargav can you check once?

File: images/teampic/pirapuraj.jpg
Comment (sid18996) [inline images/teampic/pirapuraj.jpg:1]: @neha-bhargav 
It can be removed i guess, it is not being used anywhere?

File: _data/team_members.yml
Comment (neha-bhargav) [reply _data/team_members.yml:1]: Yes, updated the same for him.

File: images/teampic/pirapuraj.jpg
Comment (neha-bhargav) [reply images/teampic/pirapuraj.jpg:1]: Yes, removed.

**Pull request merged**